test/column/renderEditCell.test.tsx

```diff
@@ -100,15 +100,15 @@ describe('Editor', () => {
     render(<EditorTest gridRows={rows} />);
     await userEvent.click(getCellsAtRowIndex(0)[0]);
     expect(getCellsAtRowIndex(0)).toHaveLength(2);
-
     await scrollGrid({ scrollTop: 2000 });
     expect(getCellsAtRowIndex(0)).toHaveLength(1);
     expect(screen.queryByRole('spinbutton', { name: 'col1-editor' })).not.toBeInTheDocument();
+    expect(getRows()[1]).toHaveTextContent('5353');
     await userEvent.keyboard('123');
-    expect(screen.getByRole('spinbutton', { name: 'col1-editor' })).toHaveValue(1230);
-    const spy = vi.spyOn(window.HTMLElement.prototype, 'scrollIntoView');
-    await userEvent.keyboard('{enter}');
-    expect(spy).toHaveBeenCalled();
```

Comment:
Why did we remove these checks?

```ts
expect(screen.getByRole('spinbutton', { name: 'col1-editor' })).toHaveValue(1230);
```
this checked that when typing on a cell, it opens the editor and the editor receives focus, and we continue typing in the editor

```ts
    const spy = vi.spyOn(window.HTMLElement.prototype, 'scrollIntoView');
    await userEvent.keyboard('{enter}');
    expect(spy).toHaveBeenCalled();
```
this checked that we scrolled back to the cell even if we had scrolled past it (see `await scrollGrid({ scrollTop: 2000 });` above)

Maybe we need more comments.

Reply:
I think it is better to check the actual row in the view instead of spying on `scrollIntoView`